### ISSUE TITLE
Hyvä session issue

### ISSUE THREAD

**Issue by Ylmzef** [yireo/Yireo_GoogleTagManager2#263]
When I enable the module, I experience an intermittent issue where the "Your session has expired" message appears while adding products to the cart using Ajax in the Hyvä theme.

**Comment by jissereitsma** [comment]
Thanks for reporting. Unfortunately, I'm not able to reproduce this myself. This extension merely makes use of the session and it adds data to the `customerData` which is also used by Hyva. Perhaps further debugging is needed by you? For instance, is there any PHP error logged anywhere? What happens if you clear cookies? What happens if you clear local storage?

**Comment by Ylmzef** [comment]
Thank you for your reply. I’ve conducted extensive debugging on this issue, including checking configurations for Varnish, Nginx, and PHP. Unfortunately, I couldn’t find any PHP error logs or anything else that points to the root cause. I also tried clearing cookies and local storage, but the issue persists.

Interestingly, the problem only disappears when I disable the module. I’m running a multi-store setup, and other stores with the same structure don’t have this module enabled and do not exhibit this issue. This suggests that the module might be related to the problem, but I couldn’t identify why. If you have any further recommendations, I’m happy to explore them.

**Comment by jissereitsma** [comment]
No, unfortunately, nothing comes to mind. Obviously, if you are the developer of the site, you can simply debug things with the regular tools and see when what is being called. But from a distance, I wouldn't know what to look for....
